Configurations/windows-makefile.tmpl

```diff
@@ -209,6 +209,7 @@ MTFLAGS= {- join(' ', @{$config{MTFLAGS}}) -}
 AS={- $config{AS} -}
 ASFLAGS={- join(' ', @{$config{ASFLAGS}}) -}
 
+RC="{- $config{RC} -}"
 RC={- $config{RC} -}
```

Comment:
Shouldn't you remove the old RC?

Reply:
Made a mistake during merging. It has been addressed in the newer version.